examples/default-values-backend/package.json

```diff
@@ -1,7 +1,6 @@
 {
   "name": "@contentful/backend_app_demo",
   "version": "1.0.0",
-  "main": "src/index.ts",
```

Comment:
I am not sure if this field is used by `lerna.json` but parcel wouldn't work with this property set. It would throw this following error. 
 ```
💡 The "main" field is meant for libraries. If you meant to output a .ts file, 
either remove the "main" field or choose a different target name.
 ```
 If we were to change it to the built file. It would throw another error related to the target file being `html` as it tries to use that property as main target.
 ```
 Target "main" declares an output file path of "built/src/index.js" which does not match 
the compiled bundle type "html".
 ```

Reply:
Afaik lerna doesn't use this field. Also, the `examples` folder in this repo is not managed by lerna.

So, I think it's fine to drop it: https://parceljs.org/getting-started/migration/#package.json%23main